[2019-04-30 17:14:54,318] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Mixed_Failing_Demo.5b_Sense_Transformed_Data 2019-04-30T21:14:23.236810+00:00 [queued]>
[2019-04-30 17:14:54,320] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Mixed_Failing_Demo.5b_Sense_Transformed_Data 2019-04-30T21:14:23.236810+00:00 [queued]>
[2019-04-30 17:14:54,320] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 17:14:54,320] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-04-30 17:14:54,320] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 17:14:54,330] {__init__.py:1374} INFO - Executing <Task(FileSensor): 5b_Sense_Transformed_Data> on 2019-04-30T21:14:23.236810+00:00
[2019-04-30 17:14:54,330] {base_task_runner.py:119} INFO - Running: ['airflow', 'run', 'Mixed_Failing_Demo', '5b_Sense_Transformed_Data', '2019-04-30T21:14:23.236810+00:00', '--job_id', '236', '--raw', '-sd', 'DAGS_FOLDER/mixed_failing.py', '--cfg_path', '/tmp/tmp91ronjqs']
[2019-04-30 17:14:54,583] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data /home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/configuration.py:214: FutureWarning: The task_runner setting in [core] has the old default value of 'BashTaskRunner'. This value has been changed to 'StandardTaskRunner' in the running config, but please update your config before Apache Airflow 2.0.
[2019-04-30 17:14:54,583] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   FutureWarning,
[2019-04-30 17:14:54,748] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data [2019-04-30 17:14:54,747] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-04-30 17:14:54,897] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data [2019-04-30 17:14:54,897] {__init__.py:305} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/mixed_failing.py
[2019-04-30 17:14:54,938] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data [2019-04-30 17:14:54,938] {cli.py:517} INFO - Running <TaskInstance: Mixed_Failing_Demo.5b_Sense_Transformed_Data 2019-04-30T21:14:23.236810+00:00 [running]> on host localhost.localdomain
[2019-04-30 17:14:54,947] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/transformed/yelp_subset_1.csv
[2019-04-30 17:15:04,964] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/transformed/yelp_subset_1.csv
[2019-04-30 17:15:14,976] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/transformed/yelp_subset_1.csv
[2019-04-30 17:15:14,983] {__init__.py:1580} ERROR - Snap. Time is OUT.
Traceback (most recent call last):
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 117, in execute
    raise AirflowSensorTimeout('Snap. Time is OUT.')
airflow.exceptions.AirflowSensorTimeout: Snap. Time is OUT.
[2019-04-30 17:15:14,987] {__init__.py:1611} INFO - Marking task as FAILED.
[2019-04-30 17:15:15,016] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data Traceback (most recent call last):
[2019-04-30 17:15:15,016] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/bin/airflow", line 32, in <module>
[2019-04-30 17:15:15,016] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data     args.func(args)
[2019-04-30 17:15:15,016] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-04-30 17:15:15,016] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data     return f(*args, **kwargs)
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 523, in run
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data     _run(args, dag, ti)
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 442, in _run
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data     pool=args.pool,
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/db.py", line 73, in wrapper
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data     return func(*args, **kwargs)
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data     result = task_copy.execute(context=context)
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 117, in execute
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data     raise AirflowSensorTimeout('Snap. Time is OUT.')
[2019-04-30 17:15:15,017] {base_task_runner.py:101} INFO - Job 236: Subtask 5b_Sense_Transformed_Data airflow.exceptions.AirflowSensorTimeout: Snap. Time is OUT.
[2019-04-30 17:15:19,364] {logging_mixin.py:95} INFO - [2019-04-30 17:15:19,364] {jobs.py:2562} INFO - Task exited with return code 1
